### PR TITLE
created : guide page

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
GuidePage
I have added the GuidePage and  also added  path to the files  . So ignore other files. Focus only on the guide.js(in pages , in guideline-page folder )

Files to check
src/Pages/guideline-page/Guide.jsx
src/Pages/guideline-page/Guide.css

PS : you can check -
Responsiveness
Navbar functionalities

image

### CODE REVIEW

File: src/pages/guideline-page/Guide.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          <li>Each right answer scores 10 Points</li>
+          <li> Each multiple choice question has only one correct answer</li>
+          <li> To win the quiz you need to score more than 70%</li>
+          <li>You can attempt each question only once.</li>
+          <li> No negative marking for wrong answers.</li>
+          <li>Question are Multiple Choice Questions.</li>
+          <li>Don't plagarize. Try to answer on your own.</li>
+          <li>You can take the quiz multiple times.</li>
+       </ul>
+        <button className='start-play-quiz'>
```
Comment (Muzammil-khan710) [inline src/pages/guideline-page/Guide.jsx:26]: Suggestion: You can remove button and put style to Link tag only

**Review by Muzammil-khan710 — APPROVED** [review]
Code is clean !
Ui is awesome
Gave a suggestion above
Overall it is good to merge!

File: src/pages/guideline-page/Guide.jsx (hunk 1 above)
Comment (Shubhamkr2610) [reply src/pages/guideline-page/Guide.jsx:26]: okay ... will remove it

**Review by arjun-computer-geek — APPROVED** [review]
UI looks good

**Pull request merged**